action/action-dsl/src/main/java/com/powsybl/action/dsl/afs/ActionScriptBuilder.java

```diff
@@ -69,6 +69,8 @@ public ActionScript build() {
             throw new UncheckedIOException(e);
         }
 
+        context.getStorage().consistent(info.getId());
```

Comment:
setConsistent(info.getId()) ? @mathbagu ?

Reply:
I agree, `setConsistent` for setters, instead of just `consistent`.